test/py/test_migration.py

```diff
@@ -1,7 +1,7 @@
 #
-# Copyright (c) 2021 Nutanix Inc. All rights reserved.
+# Copyright (c) 2023 Nutanix Inc. All rights reserved.
 #
-# Authors: Thanos Makatos <thanos@nutanix.com>
+# Authors: William Henderson <william.henderson@nutanix.com>
```

Comment:
add, don't replace, authors please.

Reply:
Whoops - I rewrote the whole file from scratch so I didn't realise it was going to look like I got rid of Thanos! I'll fix that.